cypress/plugins/index.js

```diff
@@ -34,11 +35,9 @@ module.exports = (on, config) => {
           postedAt: now,
           lastSubthreadActivity: now,
         }));
-      MongoClient.connect("mongodb://localhost:27017", async function(err, client) {
-        if(err) {
-          console.error(err);
-          return;
-        }
+      const client = new MongoClient(process.env.TESTING_DB_URL);
```

Comment:
I think I'd prefer to have this take in a command line argument and you could see if the argument was a filename or a url. That way we could save with the cross-env stuff below. My uncertainty is whether:

```
  start: yarn ea-start-testing-db --testing-db $TESTING_DB_URL
```

would work in cypress.yaml.